upup/models/cloudup/resources/addons/cluster-autoscaler.addons.k8s.io/k8s-1.15.yaml.template

```
@@ -344,6 +344,7 @@ spec:
             {{ end }}
             - --ignore-daemonsets-utilization={{ .IgnoreDaemonSetsUtilization }}
             - --scale-down-utilization-threshold={{ .ScaleDownUtilizationThreshold }}
+            - --skip-nodes-with-custom-controller-pods={{ .SkipNodesWithCustomControllerPods }}
```

I think this was added in CAS 1.27 as part of https://github.com/kubernetes/autoscaler/commit/ff6fe5833d522d15670084bf996b3dea88231986.
```suggestion
            {{ if IsKubernetesGTE "1.27.0") }}
            - --skip-nodes-with-custom-controller-pods={{ .SkipNodesWithCustomControllerPods }}
            {{ end }}
```

---

Right, it's a recent addition.

---

I think there's a typo `)` and `hack/update-expected` should run again.

---

Yes true. Is it ok to overwrite the commit with a typo with a forced push or it'll be squashed on merge anyway? What's the accepted practice about this @hakman ?

---

Please squash everything.

---

...ok, squashed, force pushed, let's try again :)